src/test/java/org/apache/commons/lang3/time/DurationFormatUtilsTest.java

```diff
@@ -473,6 +473,21 @@ public void testFormatPeriod() {
         assertEquals("helloworld", DurationFormatUtils.formatPeriod(time1970, time, "'hello''world'"));
     }
 
+    @Test
+    public void testSingleQuotes() {
+        final Calendar cal1970 = Calendar.getInstance();
+        cal1970.set(1970, Calendar.JANUARY, 1, 0, 0, 0);
+        cal1970.set(Calendar.MILLISECOND, 0);
+        final long time1970 = cal1970.getTime().getTime();
+        long time = time1970 + 60 * 1000;
+        // empty quoted strings: '' should now generate a single quote
```

Comment:
OK, now I see why I was confused. Two single quotes is **not** or no longer is an empty quoted string. This is an escape sequence for a single quote. Rewrite comment to clarify.